haskell/src/Monocle/Backend/Queries.hs

```diff
@@ -127,8 +131,8 @@ changeState state' =
   ]
 
 -- | Add a document type filter to the query
-documentType :: Text -> BH.Query
-documentType type' = BH.TermQuery (BH.Term "type" type') Nothing
+documentType :: NonEmpty Text -> BH.Query
+documentType = BH.TermsQuery "type"
```

Comment:
Not sure that makes a difference, but we could keep the TermQuery when the list only has a single element.

Reply:
It make be the OR for you if needed. I'm in favor of it